src/client/controllers/email-sign-up-controller.ts

```diff
@@ -0,0 +1,13 @@
+import { Controller } from 'stimulus';
+
+export default class EmailSignUpController extends Controller {
```

Comment:
Unless I'm missing something this doesn't yet seem to add the email to the mailing list.  Also, could we use a flash rather than `alert`?

Reply:
The submission of the form using AJAX is handled by `rails-ujs` (using `data-remote=true` in the HTML). The controller then only needs to deal with the response. I'd like to avoid a normal dialogue box: my plan is to have a common way of handling all dialogue boxes, since `rails-ujs` can generate them for us neatly with the `data-confirm` attribute (e.g. for dangerous actions, such as `DELETE` requests). But I think that would be better in a new PR, to appear before Thursday.

Reply:
Having a standard way of handling dialogue boxes sounds great.  Also, implementing the email handling in this controller would keep it in a more obvious place than `src/server/routes/api-router.ts`.